lib/ItemRow/ItemRowName.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { node, oneOfType, string } from 'prop-types';
+
+const ItemRowName = ({ children, ...props }) => {
+  return (
+    <div className="item-row__name" {...props}>
+      {children}
```

Comment:
This can be text but ideally, the consuming parent will define the correct text node e.g. `p`. Maybe I should add that to the story?

Reply:
👍